core/sql/sqlcomp/nadefaults.cpp

```diff
@@ -1759,6 +1763,7 @@ SDDkwd__(EXE_DIAGNOSTIC_EVENTS,		"OFF"),
  // exposure.
  DDkwd__(HBASE_DELETE_COSTING,		             "ON"),
  DDflt0_(HBASE_DOP_PARALLEL_SCANNER,             "0."),
+ DDkwd__(HBASE_ESTIMATE_ROW_COUNT_VIA_COPROCESSOR,   "OFF"),
```

Comment:
No. In my tests, there was a small performance penalty for using the coprocessor. We must have the coprocessor if HBase encryption is being used, but otherwise we can just use the client code.

Your next question might be: why is there a performance penalty? I looked into that a little bit. I think it is because we do dynamic coprocessor loading. It may be that the penalty is not real: this coprocessor is packaged with the DTM coprocessors. I think they all get loaded together. Assuming this is true, we are simply paying this penalty earlier (at estimate row count time) rather than at transaction start time.